### PR TITLE
IS-13381: DTL navigation menu sorted alphabetically

### PR DESCRIPTION
**Pull request by kruand-sesam**
Origin:
https://sesam.slack.com/archives/C02F37XJ7T9/p1667557640177409

YouTrack ticket:
https://sesam.myjetbrains.com/youtrack/issue/IS-13381/Sort-navigation-menu-alphabetically

### CODE REVIEW

**Comment by kruand-sesam** [comment]
It looks like Git is confused about the changes - to be clear I have only moved sections up and down, there're no changes to content.

**Comment by kruand-sesam** [comment]
@djalil-bouhachi In my opinion the code font is much prettier than removing it.
Is there no other way to impact the font size of the right navigation bar?
code-indication

File: data-transformation-language.rst [hunk 1]
```diff
@@ -75,6 +74,7 @@
    DTL: Paths <dtl/dtl-functions-paths.rst>
    DTL: Sets <dtl/dtl-functions-sets.rst>
    DTL: Strings <dtl/dtl-functions-strings.rst>
+   DTL: Transforms <dtl/dtl-functions-transforms.rst>
```
Comment (grove) [inline data-transformation-language.rst:77]: I think "DTL: Transforms" should stay at the top as they are of a different kind.

Comment (djalil-bouhachi) [reply data-transformation-language.rst:77]: Agree on that

Comment (kruand-sesam) [reply data-transformation-language.rst:77]: I'll adjust this when the other comment is clarified.

**Comment by grove** [comment]
Why are we making a font-related change? The PR is about sorting them alphabetically. I have not strong opinion on the font issue other than I think it is fine as-is.

**Comment by kruand-sesam** [comment]
Since there're no response in the PR I will revert @djalil-bouhachi 's change about font size and keep the original scope of the task in place.

Feel free to create a separate task about the font changes (@djalil-bouhachi )

**Review by grove — APPROVED** [review]

**Pull request merged**